config.json

```diff
@@ -1296,13 +1296,13 @@
         "player_warp_energy_multiplier": 0.12,
         "use_warp_energy_for_cloak": true,
         "warp_energy_multiplier": 0.12,
-        "warp_memory_effect": 0.9,
+        "warp_memory_effect": 0.5,
         "warp_multiplier_max": 300000000.0,
         "warp_multiplier_min": 1.5,
         "warp_ramp_down_time": 0.5,
         "warp_ramp_up_time": 5.0,
         "warp_stretch_cutoff": 500000.0,
-        "warp_stretch_decel_cutoff": 500000.0,
+        "warp_stretch_decel_cutoff": 2500000.0,
```

Comment:
This change reduces the amount by which the autopilot overshoots the target, with stations, for example. Now, if it overshoots at all, it’s frequently only by 5 km or so, instead of 50,000 or more.